tests/test_dataset.py

```diff
@@ -487,3 +490,65 @@ def test_sanitized_dimensions(spark_context):
 
     summaries = dataset.summaries(spark_context)
     assert len(summaries) == 2
+
+
+@pytest.fixture
+def setupDF():
```

Comment:
Fixtures are usually named as a noun, as opposed to a verb. A good name would be `dataset`.